lib/sync-settings.coffee

```diff
@@ -38,6 +50,10 @@ module.exports =
       "snippets.cson":
         content: @fileContent atom.config.configDirPath + "/snippets.cson"
 
+    for file in atom.config.get 'sync-settings.extraFiles'
```

Comment:
The tests [fail](https://travis-ci.org/Hackafe/atom-sync-settings/builds/52930584#L176). It seems on clean install the value is undefined.